src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  let stateCopy = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        stateCopy = {
+          ...stateCopy, ...action.extraData,
+        };
+        break;
+
+      case 'removeProperties':
+        stateCopy = { ...stateCopy };
```

Comment:
I think you do not need it here

Reply:
I tried to remove, but then  tests don't work

Reply:
and I don't know how to fix it

Reply:
Interesting, I'll check it out for myself later, whatever, good job )